engine/runtime-parser/src/main/java/org/enso/compiler/core/TreeToIr.java

```diff
@@ -1065,7 +1065,7 @@ loc, meta(), diag()
         yield new Name.MethodReference(
                 Option.empty(),
                 methodName,
-                methodName.location(),
+                getIdentifiedLocation(autoscopedIdentifier),
```

Comment:
Integrating Dmitry's test revealed that `..A` location didn't include the two dots. Fixed in d4392179d2960beaa7b9675bb64dc24205010fbe